telemetry/csharp/client-generator-inputs/telemetry-2017-07-25.customizations.json

```diff
@@ -0,0 +1,3 @@
+{
+    "file-description": "This file is used by the SDK Client Generator. See telemetry/csharp/README.md for details"
+}
```

Comment:
Adding a comment on ` telemetry/csharp/toolkit-telemetry.snk` is impossible, but do we need this file?

Reply:
Yup, it allows the compiled assembly to be "strongly typed", which is necessary when using it in the toolkit. Which reminds me, I will update the PR with how I verified the produced assembly.